nextflow_schema.json

```diff
@@ -788,6 +788,13 @@
                     "hidden": true,
                     "fa_icon": "fas fa-users-cog"
                 },
+                "test_data_base": {
+                    "type": "string",
+                    "default": "https://raw.githubusercontent.com/nf-core/test-datasets/sarek3",
+                    "description": "Base path / URL for data used in the test profiles",
+                    "help_text": "Warning: The `-profile test` samplesheet file itself contains remote paths. Setting this parameter does not alter the contents of that file.",
```

Comment:
Maybe a bit better description on why you might use this option? I know it's mostly for the CI, but still, users will see this.

Reply:
It's an hidden params with a default value. I think it should be fine